### PR TITLE
Add Create Dictionary to DictionaryWidget

### PR DESCRIPTION
**Pull request by iangermann**
Fixes issue #24

### CODE REVIEW

**Comment by panathea** [comment]
Learned about internationalization in Plover tonight. Can you update the strings that aren't handled properly in the code as I did here:

https://github.com/Germanika/plover/pull/28/commits/4fa83776be7217b98bff0e185dc76ea0d8deaa40

File: plover/gui_qt/dictionaries_widget.py [hunk 1]
```diff
@@ -302,6 +306,38 @@ def _open_file_directory(self):
...
+            file_directory = os.path.dirname(new_filename)
+            new_ext = os.path.splitext(new_filename)[-1].lower()
+
+            try:
+                open(new_filename, 'w')
+                dictionary = create_dictionary(new_filename)
+                dictionary.save()
+            except Exception:
+                log.error(
+                    'Error while saving new dictionary: %s',
```
Comment (panathea) [inline plover/gui_qt/dictionaries_widget.py]: Could I poke you to update this so it's translatable?

**Comment by iangermann** [comment]
I think I've updated the missing strings, but I'm not sure how to go about adding French translations. I'll probably push another update to add those in tonight

**Comment by panathea** [comment]
Don't worry about the translation file :)

**Comment by iangermann** [comment]
It's been a few years.. This isn't super relevant anymore 😀

**Pull request closed without merging**